integration-test/list_test.go

```diff
@@ -1,12 +1,27 @@
 package main
 
 import (
+	"encoding/json"
+	"strings"
 	"time"
 
 	"github.com/docker/docker/pkg/integration/checker"
 	"github.com/go-check/check"
 )
 
+type cList struct {
+	// ID is the container ID
```

Comment:
Is this structure from containerState of runc?
I suggest we use State structure from oci. Create new structure combinded with the different fields and State.
Such as
```
type containerState struct {
    specs.State
    Bundle string `json:"bundle"` 
}
```